server/apps/immich/src/api-v1/user/dto/create-user.dto.ts

```diff
@@ -1,27 +1,28 @@
+import { ApiProperty } from '@nestjs/swagger';
 import { IsNotEmpty, IsOptional } from 'class-validator';
 
 export class CreateUserDto {
   @IsNotEmpty()
+  @ApiProperty({ default: 'testuser@email.com' })
```

Comment:
This one and password, firstName, lastName, should use `example` instead of `default`